cocos/2d/CCSpriteFrameCache.cpp

```diff
@@ -61,6 +63,11 @@ SpriteFrameCache* SpriteFrameCache::getInstance()
     return _sharedSpriteFrameCache;
 }
 
+void SpriteFrameCache::SetSpriteFrameNameGenerator(SpriteFrameNameGenerator generator)
+{
+    SpriteFrameCache::_spriteFrameNameGenerator = generator;
```

Comment:
Okey... http://www.stroustrup.com/4th.html

Reply:
Lol you are so funny!